tinker/templates/admin/redirects/home.html

```diff
@@ -13,6 +13,15 @@
                 <div class="card">
                     <div class="content">
                         <form action="" id="new-redirect-form">
+                            <div class="row">
+                                <p>New or updated redirects become active automatically every 30 minutes.</p>
+                                <p>If you do not wish to wait for the next automatic update, click the button below:</p>
```

Comment:
`...click the button below:` maybe add... `...click the button below to update now:`?

Reply:
Done